sessionctx/context_test.go

```diff
@@ -36,8 +32,6 @@ func TestBasicCtxTypeToString(t *testing.T) {
 		{basicCtxType(9), "unknown"},
 	}
 	for _, tt := range tests {
-		if tt.key.String() != tt.v {
-			t.Fatalf("want %s but got %s", tt.v, tt.key.String())
-		}
+		require.Equalf(t, tt.key.String(), tt.v, "want %s but got %s", tt.v, tt.key.String())
```

Comment:
```suggestion
		require.Equal(t, tt.key.String(), tt.v)
```

testify will generate awesome error message if assertion failed. Need not do it manually.